orebfuscator-plugin/src/main/java/net/imprex/orebfuscator/config/OrebfuscatorCacheConfig.java

```diff
@@ -130,6 +137,9 @@ public int maximumSize() {
 
 	@Override
 	public void maximumSize(int size) {
+		if (size < 0) {
+			throw new IllegalArgumentException("maximum size can't be negative");
```

Comment:
https://github.com/Imprex-Development/Orebfuscator/pull/39/commits/357f9e69df26bd28bcd71605dad1f8a3b15e5e01#diff-39d99666a9d85f7aef6a46839ae54113R34 you wrote lower than one but this is lower than zero